page-black-history-landing.php

```diff
@@ -4,55 +4,201 @@
 */
 get_header(); ?>
 <div class="black-history">
+	<?php while ( have_posts() ) : the_post();
+	$feature_img = wp_get_attachment_image_src( get_post_thumbnail_id( $post->ID ), 'full' );
+	if ( $feature_img ) { ?>
+	<div class="banner feature-img" role="banner" style="background: url( <?php echo $feature_img[0] ?> ) no-repeat center center;background-size: cover;">
+	<?php } else { ?>
 	<div class="banner" role="banner">
+	<?php } ?>
 		<?php get_template_part( 'breadcrumb' ); ?>
 		<div class="heading-banner text-center">
 			<div class="container">
 				<div class="row">
 					<div class="col-md-12">
-						<h1>Black History</h1>
+						<h1><?php the_title(); ?></h1>
 					</div>
 				</div>
+				<?php get_image_caption( 'top' ); ?>
 			</div>
 		</div>
 	</div>
-</div>
-<div class="menu" role="navigation">
-	<div class="container">
-		<div class="row">
-			<nav class="col-md-12 text-center">
-				<ul>
-					<li>
-						<a href="#">Menu item 1</a>
-					</li>
-					<li>
-						<a href="#">Menu item 2</a>
-					</li>
-					<li>
-						<a href="#">Menu item 3</a>
-					</li>
-				</ul>
-			</nav>
+	<div class="menu" role="navigation">
+		<div class="container">
+			<div class="row">
+				<nav class="col-md-12 text-center">
+					<ul>
+						<li>
+							<a href="#blog-0">From our blog</a>
+						</li>
+						<li>
+							<a href="#event">What's on</a>
+						</li>
+						<li>
+							<a href="#podcast-0">Podcasts</a>
+						</li>
+						<?php
+							$first_section_heading = get_post_meta( $post->ID, 'first_section_heading', true );
+							if ( $first_section_heading ) { ?>
+						<li>
+							<a href="#first-content-section"><?php echo $first_section_heading ?></a>
+						</li>
+							<?php } ?>
+					</ul>
+				</nav>
+			</div>
 		</div>
 	</div>
-</div>
-<main id="main" class="content-area" role="main">
-	<div class="container">
-		<div class="row">
-			<section>
-				<div class="col-md-12">
-					<article>
-						<div class="entry-header">
-							<h2>Black History</h2>
+	<main id="main" role="main" class="content-area">
+		<div class="container">
+			<?php if ( get_post()->post_content ) { ?>
```

Comment:
Of the two control structure syntax my understanding is that the alternative exists for circumstances like this (with the curly braces being better suited to PHP only code). Opinion is divided about it though - see http://stackoverflow.com/questions/564130/difference-between-if-and-if-endif (My initial thought is that `endif;` helps with readability but I accept Chris' point about highlighting the matching brace. 

That said, the WordPress code style doesn't dictate either https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style 

We should decide on one and use it though - for consistency across our templates. Let's decide on it as a team (I'll put something on Slack).